core/modules/qdisp/QueryRequest.cc

```diff
@@ -109,14 +112,17 @@ class QueryRequest::AskForResponseDataCmd : public PriorityCommand {
         // If more data needs to be sent, _processData will make a new AskForResponseDataCmd
         // object and queue it.
         {
+            util::InstanceCount _ic{"3-act-AskForResponseDataCmd&&&"};
```

Comment:
Same comment on the name of the local variable which should not start with underscore.